nilearn/datasets/tests/test_func.py

```diff
@@ -1066,28 +1081,46 @@ def test_fetch_spm_multimodal(tmp_path):
     assert dataset.description != ""
 
 
-def test_fetch_spm_multimodal_missing_data(tmp_path):
-    _generate_spm_multimodal(tmp_path, n_sessions=2, n_vol=390)
+def _generate_spm_multimodal_arc(n_sessions=2, n_vol=390):
```

Comment:
This function does something very similar to _generate_spm_multimodal above (line 1047).
Should we try to combine them into a single function?